spec/factories.rb

```diff
@@ -15,8 +15,8 @@
     role "sysadmin"
   end
 
-  factory :threshold_user, :parent => :admin_user do
-    role "threshold"
+  factory :moderator_user, :parent => :admin_user do
```

Comment:
Maybe just take the role out of the admin_user factory, and make sure we always use sysadmin_user, or moderator_user ?